Doppler.Currency/Factory/CurrencyFactory.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using Doppler.Currency.Enums;
+using Doppler.Currency.Services;
+
+namespace Doppler.Currency.Factory
+{
+    /// <summary>
+    /// Class for the Currency factory
+    /// </summary>
+    public class CurrencyFactory : ICurrencyFactory
+    {
+        #region Private Attributes
```

Comment:
I think that use #region is when on too big the class

Reply:
I agree: if we can avoid using `#region`, it is better.

Reply:
OK, I will remove the #region

Reply:
updated